cloudant-client/src/test/java/com/cloudant/tests/CloudantClientTests.java

```diff
@@ -105,17 +110,48 @@ public void cookieTest() {
     }
 
     // java-cloudant/n.n.n or java-cloudant/unknown followed by 4 groups of /anything
-    private final String userAgentRegex = "java-cloudant/(?:(?:\\d+.\\d+.\\d+)|(?:unknown))" +
+    private final String userAgentRegex = "java-cloudant/(?:(?:\\d+.\\d+.\\d+))" +
             "(?:/{1}[^/]+){4}";
+
+    private final String userAgentUnknownRegex = "java-cloudant/(?:(?:unknown))" +

```

Comment:
This should be `cloudant-http` instead of `java-cloudant` if we can't retrieve the user agent shouldn't it?